core/templates/services/literally-canvas-helper.service.ts

```diff
@@ -287,10 +271,7 @@ angular.module('oppia').factory('LiterallyCanvasHelperService', [
         return snapshot;
       },
 
-      isSvgTagValid: function(svgString) {
-        // This function is used to validate the svg tag.
-        return validateSVGTag(svgString);
-      },
+      isSvgTagValid: isSvgTagValid,
```

Comment:
Why don't you just keep the function definition here?

If that's not possible, then maybe prefix the inner function with _.

Reply:
As I have mentioned [here](https://github.com/oppia/oppia/pull/9445#discussion_r436263533). It causes problems when I mention it there so I have used _. Done.